client/src/App.jsx

```diff
@@ -53,9 +53,16 @@ const App = () => {
   return (
     <AuthProvider>
       <div>
+<<<<<<< HEAD
+        {/* Home Page */}
+        <Modal classname='mfn_modal'show={modalState}/>
```

Comment:
Modal and mfn button should be commented out, these are meant to be imported in individual components

Reply:
Got it I commented them out